proposals/stf-general-investment.md

```diff
@@ -0,0 +1,263 @@
+# The great fix forward – Sovereign Tech Fund general investment
+
+This proposal was collaboratively developed by
+
+- @fricklerhandwerk
+- @proofconstruction
+- @RaitoBezarius
+- @wamirez
+
+with contributions by (in ASCIIbetical order):
+
+- @Ericson2314
+- @Janik
+- @edef
+- @mweinelt
+- @refroni
+- @roberth
+- @tfc
+
+## Project title
+
+*May be identical with application name*
+
+Transforming global software distribution with Nix
+
+## Describe your project in a sentence. 100 words
+
+Nix makes computation with file systems highly repeatable; Nixpkgs is the largest, most up-to-date free software collection in the world; NixOS is a Linux distribution with a uniform declarative configuration interface.
+
+## Describe your project more in-depth. Why is it critical? 300 words
+
+[Nix](https://nix.dev) reduces the global cost of software production and long-term use by a tangible amount.
+It is the basis of an ecosystem of exceptionally powerful tools – including Nixpkgs, [the largest, most up-to-date software repository in the world](https://repology.org/repositories/graphs), and NixOS, a Linux distribution that can be configured fully declaratively, with unmatched flexibility.
+
+The Nix ecosystem is already on the way to become a default in the software development tool stack, by drastically shortening the time required to start working on software projects – written in any language – and lowering the cost of maintaining and sharing build setups and development environments.
+It offers tools to bring together all of open source in one coherent, dependable environment.
+
+It also serves as the platform to accelerate, keep usable, and drive adoption of countless software innovations that would otherwise be inaccessible to their audiences.
+Moreover, the public [Nix binary cache](http://cache.nixos.org/) provides operational software artefacts from more than a decade back, some of which cannot be obtained otherwise any more.
+It thus already supports technological sovereignty and autonomy for individuals and organisations alike.
+
```

Comment:
```suggestion

NixOS empowers the whole open source ecosystem.
A large number of open source applications and decentralized internet services has been modeled in NixOS modules, so that they can be installed with a simple line of code and configured in a consistent and coherent way.
This significantly lowers the barrier to open source adoption, and adds significant value to all these open source projects.
Furthermore it stimulates contributions to open source, by making it easy to [deploy patches](https://nixos.org/manual/nixpkgs/stable/index.html#chap-overlays) to any of these projects.

```

Reply:
Yeah I think this is an important addition; the Nix project(s) should be funded for many reasons, but one of the biggest is the positive downstream effect on usability for the rest of the open source ecosystem. Easing _contribution_ is noted elsewhere in the document.

Minor wording changes:

``` diff
- has been modeled in NixOS modules
+ have been modeled with NixOS modules
```
`has -> have` is just a typo fix for verb-tense agreement, but the `in -> with` change is more debatable; I suggest this one to keep with the most common usage, though the logician in me prefers the former. :slightly_smiling_face: 

``` diff
- simple line of code
+ single line of code
```
Here I'm assuming you're referring to being able to set `<thing>.enable = true;` and have intended things happen. Simplicity is another matter entirely (don't look behind the curtain!)

Reply:
Well put.